index.js

```diff
@@ -831,6 +906,15 @@ module.exports.bool = {
   or: 'or',
   eor: 'eor'
 };
+// Colourspaces
+module.exports.colourspace = {
+  multiband: 'multiband',
+  'b-w': 'b-w',
+  bw: 'b-w',
+  cmyk: 'cmyk',
+  srgb: 'srgb'
+};
```

Comment:
The trouble with other colorspaces supported by vips is that its hard to write images in those formats. You can always use the string nick to access them, but by not putting them in the `colourspace` enum it signals that they aren't nicely "supported". Happy to add them if you feel otherwise...